src/storage/conversion/TypedRepresentationConverter.ts

```diff
@@ -0,0 +1,18 @@
+import { RepresentationConverter } from './RepresentationConverter';
+
+/**
+ * A {@link RepresentationConverter} that allows requesting the supported types.
+ */
+export abstract class TypedRepresentationConverter extends RepresentationConverter {
+  /**
+   * Get a hash of all supported input content types for this converter, mapped to a numerical priority.
+   * @returns A promise resolving to a hash mapping content type to a priority number.
```

Comment:
Might be good to document here that this priority number should be a value between 0 and 1.

Reply:
True, will change